tools/oeedger8r/src/Sources.ml

```diff
@@ -945,6 +944,10 @@ let generate_untrusted (ec : enclave_content) (ep : Intel.Util.edger8r_params) =
     "#include <string.h>";
     "#include <wchar.h>";
     "";
+    "#if defined(__GNUC__)";
```

Comment:
Why does oeedger8r need to generate this?

Reply:
In order to publish the internal edl files we will need to get rid of the _wrapper.c files. This means everything they do must now be done in the generated code.

The generated code doesn't forward-declare its functions which throws a warning on GCC which we treat as an error. So we can either forward declare the functions or add this. I'm fine with either.

Reply:
Ah ok. Can we make that function static instead?

Reply:
In #2609 I have tweaked oeedger8r to generate 'static' keyword for local functions and removed these  workarounds in _wrapper.c files.

Reply:
Thanks. Will remove in the next iteration.